lib/elixir/lib/dict.ex

```diff
@@ -872,6 +872,14 @@ defmodule Dict do
   @doc """
   Returns a list of key-value pairs stored in `dict`.
   No particular order is enforced.
+  
+  ## Examples
+
+      iex> dict = dict_impl.new
+      iex> dict = Dict.put(dict, :a, 1)
+      iex> dict = Dict.put(dict, :b, 2)
+      iex> Dict.into_list(dict)
```

Comment:
The issue with this test is that it assumes ordering... and we cannot do it for dicts.